integrations/access/accessrequest/message.go

```diff
@@ -134,10 +146,20 @@ func MsgReview(review types.AccessReview) (string, error) {
 	return builder.String(), nil
 }
 
-func msgFieldToBuilder(b *strings.Builder, field, value string) {
+func msgFieldToBuilder(b *strings.Builder, field, value string, additionalFields ...string) {
```

Comment:
We expect `key:value` pairs, so we might as well send a `map[string]string` instead of using a variadic function

Reply:
Hmm, we call this helper a number of times. Might not be ideal to construct a new map each time.